### PR TITLE
Mobile Portfolio setup

### PR DESCRIPTION
**Pull request by Johnadibe**
- Built with CSS Grid
- Built Toolbar and Headline Section
- I used the required fonts
- Exported images from the Figma Template

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Mobile Portfolio Setup</title>
+</head>
+
+<body>
+    <header class="header padding-left-right">
+        <h2 class="logo">John Doe</h2>
```
Comment (CollinsTatang) [inline index.html]: ### **Nice Job so far, you are doing great**

- [ ] Kindly add a link to the logo with an `a tag`, this is for three reasons: convention, semantics, and user experience(UX).
[Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+                    <a href="#">
+                        <img class="icon" src="img/icon-5.svg" alt="mercedes social icon" />
+                    </a>
+                </div>
+            </div>
+
+            <div class="headline-img-box">
+                <img class="headline-img" src="img/headline-img.svg" alt="vector graphic image" />
+            </div>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please use CSS background-image (decorations are always background images, not `img` tags). 

- If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info

File: index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                    </a>
+
+                    <a href="#">
+                        <img class="icon" src="img/icon-4.svg" alt="twitter social icon" />
+                    </a>
+
+                    <a href="#">
+                        <img class="icon" src="img/icon-5.svg" alt="mercedes social icon" />
+                    </a>
+                </div>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Please wrap all your social links in `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)
- Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Johnadibe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @Johnadibe 
, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions
- [ ] Align the icons properly
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Johnadibe) [reply index.html]: Okay Thank you @CollinsTatang 😊

File: index.html (hunk 2 above)
Comment (Johnadibe) [reply index.html]: Acknowledged. I will do that right away☺

File: index.html (hunk 3 above)
Comment (Johnadibe) [reply index.html]: Okay Thank you @CollinsTatang😊

**Comment by Johnadibe** [comment]
Thanks😊

**Pull request merged**